Rust/rust/docs/quickstart.md

```diff
@@ -81,6 +81,24 @@ cd rust/samples/HelloWorld/hello_world_high_level_app
 
     The device's led light should also blink on and off in 1 second cycles.
 
+## Build, Debug and Deploy using Visual Studio Code
+
+1. Navigate to one of the examples in the `samples` directory from the command line:
+
+```bash
+cd rust/samples/HelloWorld/hello_world_high_level_app
+```
+
+1. Open Visual Studio Code:
+
+```bash
+code .
+```
+
+1. Set `program` to the executable's full path inside the configuration in `.vscode/launch.json`. The executable's path will be located at `rust/target/armv7-unknown-linux-musleabihf/debug/hello_world_high_level_app`. 
```

Comment:
Can you add `program`?

Reply:
Is this still needed?  The sample should work without `program` here.

Reply:
This portion is unnecessary after the latest changes. Thanks for the feedback.